php/example_code/dynamodb/Readme.md

```diff
@@ -0,0 +1,84 @@
+# Amazon DynamoDB code examples for the AWS SDK for PHP v3
```

Comment:
DynamoDB code examples for the SDK for PHP (v3)

Reply:
Fixed.